src/features/currentTodo.ts

```diff
@@ -1,10 +1,17 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
```

Comment:
typing initialState can make the code more readable

```suggestion
type CurrentTodoState = Todo | null;

const initialState: CurrentTodoState = null;
```